### ISSUE TITLE
Make queries based on the FIPS codes in the product API

### ISSUE THREAD

**Issue by weikang9009** [cenpy-devs/cenpy#93]
It would be nice to be able to make queries based on the FIPS codes (state, MSA, county, etc) in the product API:
![image (7)](https://user-images.githubusercontent.com/7359284/72196461-0e553980-33cd-11ea-94b9-5c5bd3641c64.png)

**Comment by knaaptime** [comment]
I agree that would definitely be useful. One question is, would you want that implemented for *all* geographies?

For states, it would be pretty straightforward since we already have the [lookup table](https://github.com/cenpy-devs/cenpy/blob/master/cenpy/stfipstable.csv) in the repo and could just replace the code with the name before sending to the census api. But to support more geographies we'd need to add a [larger table](https://github.com/walkerke/tidycensus/blob/master/data/fips_codes.rda) like tidycensus or try and grab a current version from census on the fly.

Porting that over would probably cover most use cases but would we also want to support, e.g. `place`  and `cbsa` fips?

**Comment by ljwolf** [comment]
`cenpy.explorer.fips_table()` should be able to grab place & subcounty. But, CBSA would have to be added from a different location?

**Comment by knaaptime** [comment]
`geosnap` uses the one from [here](https://www.census.gov/geographies/reference-files/time-series/demo/metro-micro/delineation-files.html), though there might be something easier
